config.json

```diff
@@ -1,14 +1,14 @@
 {
-  "peer_addrs": ["peer0.org1.example.com:7051","peer0.org2.example.com:9051"],
+  "peer_addrs": ["peer0.org1.example.com:7051", "peer0.org2.example.com:9051"],
   "orderer_addr": "orderer.example.com:7050",
   "channel": "mychannel",
   "chaincode": "mycc",
   "version": "",
-  "args": ["query","a"],
+  "args": ["put","a","10"],
```

Comment:
I don't like `put,a,10` considering with mvcc, or say avoid conflicts. how about `get,a`?

Reply:
there's no conflict if we don't read.. and plain read doesn't really stress peer that much

Reply:
btw, i updated sample cc in a previous commit

Reply:
https://github.com/guoger/stupid/blob/master/chaincodes/putget.go#L19
you sure no mvcc conflicts? if no conflicts, it's fine.